### PR TITLE
feat(core): add db configuration command

### PR DESCRIPTION
**Pull request by dated**
## Summary

Adds a new config command to the cli so that the database can be easily configured.

```
ark config:database --host=localhost --port=5432 --username=ark --database=ark_mainnet --password=password
```

Configuring the database credentials is possible already through the cli by setting the respective environment variables. This however requires knowledge over these variable names.

## What kind of change does this PR introduce?

- [ ] Bugfix
- [x] New feature
- [ ] Refactoring / Performance Improvements
- [ ] Build-related changes
- [ ] Documentation
- [ ] Tests / Continuous Integration
- [ ] Other, please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Does this PR release a new version?

- [ ] Yes
  - [ ] All tests are passing
  - [ ] All benchmarks are passing without any _major_ regressions
  - [ ] Sync from 0 works on mainnet
  - [ ] Sync from 0 works on devnet
  - [ ] Starting a new network and forging on it work
  - [ ] Explorer is fully functional
  - [ ] Wallets are fully functional
- [x] No

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation _(if appropriate)_

## Other information

The CLI does not interact with the database in any way; changing the database name for instance will not rename the postgres database itself.

### CODE REVIEW

File: packages/core/src/commands/config/db.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+`,
+        `Set the database name
+$ ark config:db --database=ark_mainnet
+`,
+        `Set the database password
+$ ark config:db --password=password
+`,
+    ];
+
+    private static readonly validFlags: string[] = ["host", "username", "database", "password"];
```
Comment (faustbrian) [inline packages/core/src/commands/config/db.ts]: Any reason you didn't include a `port` flag?

Comment (dated) [reply packages/core/src/commands/config/db.ts]: I went by the vars laid out in https://docs.ark.io/guidebook/core/configuration.html#environment-configuration, i'll check up on the port envvar.

Comment (dated) [reply packages/core/src/commands/config/db.ts]: I'll add it here and to the docs asap.

File: packages/core/src/commands/config/db.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+`,
+        `Set the database name
+$ ark config:db --database=ark_mainnet
+`,
+        `Set the database password
+$ ark config:db --password=password
+`,
+    ];
+
+    private static readonly validFlags: string[] = ["host", "port", "username", "database", "password"];
```
Comment (supaiku0) [inline packages/core/src/commands/config/db.ts]: `database` should be `name` or vice versa?

File: packages/core/src/commands/config/db.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        host: flags.string({
+            description: "the host of the database"
+        }),
+        port: flags.integer({
+            description: "the port of the database"
+        }),
+        username: flags.string({
+            description: "the name of the database user",
+        }),
+        name: flags.string({
```
Comment (supaiku0) [inline packages/core/src/commands/config/db.ts]:
```suggestion
        database: flags.string({
```

**Pull request merged**